android/apollo/src/com/andrew/apollo/ui/fragments/profile/ApolloFragment.java

```diff
@@ -149,6 +152,13 @@ protected void onSongItemClick(int position) {
     protected ApolloFragment(int groupId, int loaderId) {
         this.GROUP_ID = groupId;
         this.LOADER_ID = loaderId;
+        this.mDefaultFragmentEmptyString = R.string.empty_music;
```

Comment:
reuse the explicit constructor:
`this(groupId, loaderId, R.string.empty_music)`